src/mapml/handlers/ContextMenu.js

```diff
@@ -220,6 +235,22 @@ export var ContextMenu = L.Handler.extend({
     context._copyData(mapEl.outerHTML.replace(/<div class="mapml-web-map">.*?<\/div>|<style>\[is="web-map"].*?<\/style>|<style>mapml-viewer.*?<\/style>/gm, ""));
   },
 
+  // Add support for pasting GeoJSON in the future
+  _pasteLayer: function(e){
+    let context = e instanceof KeyboardEvent ? this._map.contextMenu : this.contextMenu,
+      mapEl = e instanceof KeyboardEvent?this._map.options.mapEl:this.options.mapEl;
+    navigator.clipboard
```

Comment:
I see you used the clipboard api here, which is great. No need to go back and change the _copyData!  Like a light switch in an old house, the best testing is 30 years of usage